sphinx/builders/epub.py

```diff
@@ -188,6 +188,8 @@ class EpubBuilder(StandaloneHTMLBuilder):
     add_permalinks = False
     # don't add sidebar etc.
     embedded = True
+    # disable download role
+    download = False
```

Comment:
`download` is verb, so it is not good for boolean variable to me.
What about `downloadable` or `support_download` instead?